app/controllers/application_controller.rb

```diff
@@ -26,10 +26,25 @@ def configure_permitted_parameters
     end
   end
 
+  def check_owner
+    item = model.find(params[:id])
```

Comment:
Similarly, could we call this check_permissions? Since again, owner suggests ownership rather than permissions to manipulate/modify/edit/delete/what have you.